### DISCUSSION TITLE
ClassID not getting resolved facing issue from example given

### DISCUSSION THREAD

**Discussion by rajshaswat** [digitalpetri/ethernet-ip#57] [General]
Hi Kevin
i am trying to use the library and use the example given by you
i am able to connect with the device.
but in below code CalssID is not getting resolved is any changes is required?
GetAttributeListService service = new GetAttributeListService(
new PaddedEPath(new ClassId(0x01), new InstanceId(0x01)),
new int[]{4},
new int[]{2}
);
![image](https://user-images.githubusercontent.com/51187529/175809628-889442fc-e68f-4c99-83e5-5f61f1d8e088.png)

**Comment by kevinherron** [comment]
It just looks like you don’t have the appropriate import statement?

**Reply by rajshaswat** [reply]
i have below import 
![image](https://user-images.githubusercontent.com/51187529/176358460-385a5ac9-fe9f-49b7-a976-8915d832cfea.png)

can you tell me which one is missing?

**Reply by rajshaswat** [reply]
Thanks Kevin 
i got the Import.
